optuna/storages/_redis.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, url: str) -> None:
 
         _imports.check()
 
+        self._url = url
```

Comment:
In `RDBStororage`, `url` is a public attribute. Therefore I'm also wondering I should make it public.
https://github.com/optuna/optuna/blob/20ad7140c1bf492c45cb056593eecf3af4fee77f/optuna/storages/_rdb/storage.py#L104

Reply:
If we provide the `get_url` method, I think `_url` can be private so that we can encourage users to use the method instead of the property.